src/tests/suite/daos_obj_ec.c

```diff
@@ -450,6 +450,27 @@ trigger_and_wait_ec_aggreation(test_arg_t *arg, daos_obj_id_t *oids,
 	daos_debug_set_params(arg->group, -1, DMG_KEY_FAIL_LOC, 0, 0, NULL);
 }
 
+void
+trigger_and_wait_ec_aggreation_2dkeys(test_arg_t *arg, daos_obj_id_t *oids,
+				      int oids_nr, char *dkey, char *dkey2, char *akey,
+				      daos_off_t offset, daos_size_t size)
+{
+	int i;
+
+	daos_debug_set_params(arg->group, -1, DMG_KEY_FAIL_LOC,
+			      DAOS_OBJ_EC_AGG_LEADER_DIFF | DAOS_FAIL_ALWAYS, 0, NULL);
+
+	print_message("wait for 30 seconds for EC aggregation.\n");
+	sleep(30);
```

Comment:
Looks old tests used FORCE_EC_AGG fault injection to reduce test time, any reason we replace it with sleep(30)?

Reply:
old test use FORCE_EC_AGG also with sleep(30), FORCE_EC_AGG flag seems not really useful.
And this test case need another fail loc on server side DAOS_OBJ_EC_AGG_LEADER_DIFF, seems cannot set two fail loc at the same time, so I did not use FORCE_EC_AGG

Reply:
I see, I was thinking the FORCE_EC_AGG is for reducing test time. As for setting two fail locs, we could change aggregation code to force aggregation for you new fail loc (similar to FORCE_EC_AGG).

Reply:
the ci test report an failure, should be just the ec agg not finished yet. 
I'll change as what you suggested force agg for the new fail loc